lib/root_controller.js

```diff
@@ -0,0 +1,174 @@
+var Tessel = require('./tessel/tessel');
+var logs = require('./logs');
+var debug = require('debug')('controller');
+
+// create an object for t2 root steps for beeing more handy about unit testing of the T2 root command itself.
+var controller = {};
+controller.ssh = {
+  makeMenu: function(tessel, index, rtm) {
+    if (tessel.connection.connectionType === 'LAN') {
+      if (!tessel.connection.authorized) {
+        rtm.add(index + ') ' + tessel.name + ': ' + tessel.connection.ip + ' (not authorized) \n');
+      } else {
+        rtm.add(index + ') ' + tessel.name + ': ' + tessel.connection.ip + ' (authorized) \n');
+      }
+
+    } else if (tessel.connection.connectionType === 'USB') {
+      if (!tessel.connection.authorized) {
+        rtm.add(index + ') ' + tessel.name + ': [USB] (not authorized) \n');
+      } else {
+        rtm.add(index + ') ' + tessel.name + ': [USB] (authorized) \n');
+      }
+    } else {
+      rtm.add('EXIT');
+    }
+    // FIXME: Workaround for trouble with dynamic added menu elements
+    rtm._draw();
+  },
+  runSSH: function(id, opts, tessels, resolve, reject) {
+    // clear console
+    debug('runSSH...\n root@' + tessels[id].connections[0].host);
+    if (opts.menu) {
+      // because the test needs to resolve the promise ...
+      resolve({
+        opts: opts,
+        tessels: tessels
+      });
+    } else {
+      // if this is no test, starting the menu as child process
+      process.stdout.write('\u001b[2J\u001b[0;0H');
```

Comment:
`'\u001b[2J\u001b[0;0H'` occurs 6 times in this file—please store in a variable and include a comment for explanation